src/tests/ftest/util/ddb_utils.py

```diff
@@ -0,0 +1,226 @@
+"""
+  (C) Copyright 2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import os
+
+from ddb_utils_base import DdbCommandBase
+
+
+class DdbCommand(DdbCommandBase):
+    # pylint: disable=too-many-ancestors,too-many-public-methods
+    """ddb command class.
+
+    Component path is needed for most of the commands. They're in the form of:
+    [container]/[object]/[dkey]/[akey]
+
+    Each component can be indexed by UUID, but indexing is usually more convenient. e.g.,
+    "ls [0]/[1]" means index into the first container, second object, and list the dkeys
+    in it. Note that the order we add container, object, dkey/akey may not be consistent
+    with the indices, so it's better for tests to use the UUID.
+    """
+
+    def __init__(self, path, mount_point, pool_uuid, vos_file):
+        """Constructor that sets the common variables for subcommands.
+
+        Args:
+            path (str): Path to the ddb command. Pass in self.bin for our wolf/CI env.
+            mount_point (str): DAOS mount point where pool directory is created. e.g.,
+                /mnt/daos, /mnt/daos0.
+            pool_uuid (str): Pool UUID.
+            vos_file (str): VOS file name that's located in /mnt/daos/<pool_uuid>. It's
+                usually in the form of vos-0, vos-1, and so on.
+        """
+        super().__init__(path)
+        self.mount_point = mount_point
+        self.pool_uuid = pool_uuid
+        self.vos_file = vos_file
+
+        # Construct the VOS file path where ddb will inject the command.
+        self.vos_path.value = os.path.join(
+            self.mount_point, self.pool_uuid.lower(), self.vos_file)
```

Comment:
Do these need to be class attributes?  I don't see them being used anywhere else in the class other than to set `self.vos_path.value`.  It seems like  a class method to set/reset the `self.vos_path.value` would be more useful, something like:
```suggestion
        self.update_vos_path(mount_point, pool_uuid, vos_file)

    def update_vos_path(self, mount_point, pool_uuid, vos_file):
        """Update the vos_path ddb command argument.
        
        Args:
            mount_point (str): DAOS mount point where pool directory is created. e.g.,
                /mnt/daos, /mnt/daos0.
            pool_uuid (str): Pool UUID.
            vos_file (str): VOS file name that's located in /mnt/daos/<pool_uuid>. It's
                usually in the form of vos-0, vos-1, and so on.
        """
        vos_path = os.path.join(self.mount_point, self.pool_uuid.lower(), self.vos_file)
        self.vos_path.update(vos_path, "vos_path")
```